store/key-features.txt

```diff
@@ -0,0 +1,10 @@
+
+Key features at a glance:
+
+* Automatic e-mail end-to-end-encryption with Autocrypt
+* Instant chatting and synchronization across devices
+* Reach anyone out of billions of people -- just use their e-mail address!
```

Comment:
I'd actually move this aspect to the top.

Reply:
Any how about this?
```suggestion
* Chat with billions of people -- using their e-mail address!
```

Reply:
i had this first -- but do you want to chat with billions? ;)

Reply:
How about this?
```suggestion
* Chat with anyone, using their e-mail address
```